### ISSUE TITLE
Need to tell which students have submitted

### ISSUE THREAD

**Issue by teach8** [JuiceMind/JuiceMind#402]
There needs to be a quick way to see which students have submitted an assignment when viewing students on the right side panel. The color coding for passing tests is great but I also need to know when the student is done and has actually submitted the assignment. I currently can't find that info anywhere.

--------------------------------------------------------
TF: For assignments, can we have a visual, such as a green checkmark or green bar, to indicate that a student submitted an assignment? I have to click into individual pages to see if they completed their work.

**Comment by JL248** [comment]
Yes, this is really needed. Especially in situations when unit tests are not used.

**Comment by karlr-dev** [comment]
@teach8 @JL248 Thank you for the suggestion. Here's a breakdown of the coloring system we now use for students on the right.

Also, you may know this already but here's a breakdown of what all the colors mean in case you were wondering:
Grey - Student has not submitted yet
Green - Student received full credit
Red - Student received 0 credit
Orange/yellow - Student received partial credit
Blue - Student submitted but question is waiting for a grade by the teacher
